docs/actions/introduction.md

```diff
@@ -0,0 +1,171 @@
+
+!!! note
+    All actions performing a request on a remote service need to know the `Target`. While other action, like validating XML data, don't need a target.
+
+## Outputs
+
+Outputs contain the data collected after performing an action.
+These data are set in the execution context and can be accessed and used later in another action.  
+Each action provide a set of default outputs. But they are generic and may contain much more information than what you actually need.
+
+!!! note
+    The execution context holds outputs in a key/value map, where the key is a string and the value is typed.
+
+!!! warning
+    Since the execution context is a map, default outputs are overridden if you run the same action more than once in the scenario or if outputs have the same name (key).
+
+!!! important
+    We strongly recommend you to define your own outputs for setting relevant data in the execution context.
+
+### How to use outputs
+
+Let's say you are doing an HTTP GET request. By default, this action has 3 outputs: `status`, `body`, `headers`, but you want to capture a specific value from the response body.
+
+In order to do so, you need to use an [expression](/todo) and Chutney [functions](/todo), so we recommend you to read about them for further details.
+
+Let's see a simple example where you want to get a list of movie titles rated above 85/100.
+
+```json title="Response body"
+{
+    "movies": [
+        {
+            "title": "Castle in the Sky",
+            "director": "Hayao Miyazaki",
+            "rating": 78
+        },
+        {
+            "title": "Grave of the Fireflies",
+            "director": "Isao Takahata",
+            "rating": 94
+        },
+        {
+            "title": "My Neighbor Totoro",
+            "director": "Hayao Miyazaki",
+            "rating": 86
+        }
+    ]
+}
+```
+
+The best way to filter and extract the relevant data from a JSON document is to use a JSONPath expression.  
+Here is the one for our example : `$.movies[?(@.rating > 85)].title`
+
+In order to process it, you would need to write code using a JSONPath library and then tell Chutney to run your custom code.  
+Chutney provides a way to run custom code during scenario execution using [Spring Expression](/todo).
+
+Here is a raw expression you could write : `${T(com.jayway.jsonpath.JsonPath).parse(#body).read("$.movies[?(@.rating > 85)].title")}`  
+Fortunately, Chutney also provides [Functions](/todo) for common tasks, which helps you write SpEL.  
+In this case you can use the `json` function and the resulting SpEL would become : `${#json(#body, '$.movies[?(@.rating > 85)].title')}`
+
+Now that you know what this cryptic expression is and does, let's see the full HTTP GET action with outputs :
+
+=== "Kotlin"
+
+    ``` kotlin
+    HttpGetTask(
+        target = "ghibli_movie_service",
+        uri = "/all?offset=0&limit=3",
+        outputs = mapOf(
+            "bestMovies" to "${#json(#body, '$.movies[?(@.rating > 85)].title')}"
```

Comment:
do not compile. Should be : 
"bestMovies" to "\\${#json(#body, '$.movies[?(@.rating > 85)].title')}"
or
"bestMovies" to jsonPath("#body", "$.movies[?(@.rating > 85)].title")